Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:18
+
+# Set working directory
+WORKDIR /src
+
+# Install dependencies
+COPY package*.json ./
+RUN npm install
```

Comment:
`npm install` runs in the `"postCreateCommand"` of the `devcontainer`, don't need to do it twice.

Reply:
@axecopfire Good point, I'll remove the RUN npm install from the Dockerfile since it's already being executed in the postCreateCommand of the devcontainer.json.